source/community/reactnative/src/visualizer/renderer.js

```diff
@@ -48,11 +58,31 @@ export default class Renderer extends React.Component {
         this.onModalClose = props.onModalClose;
     }
 
+    bindPayloadWithData() {
+        // Create a Template instance from the template payload
```

Comment:
Under `Data Binding samples` in Visualizer, both rendered output as well as the associated JSON are same as other sample payloads - no difference to showcase the template-data binding feature. Instead of showing the parsed JSON, show the template and the data in the sample.

Reply:
Updated the JSON view with template.

Reply:
Updated the JSON view with template.